bundle/edu.gemini.programexport.servlet/src/main/scala/edu/gemini/programexport/servlet/osgi/Activator.scala

```diff
@@ -30,8 +37,29 @@ final class Activator extends BundleActivator {
   val user: JSet[Principal] = java.util.Collections.singleton[Principal](StaffPrincipal.Gemini)
 
   def start(ctx: BundleContext): Unit = {
+
+    // Extract the HTTP (internal) port number if set in the bundle property.
+    val internalPort = \/.fromTryCatchNonFatal {
```

Comment:
Is this closing the export servlet only to the internal port?

Reply:
Yes.